.github/workflows/ci.yml

```
@@ -16,3 +16,5 @@ jobs:
         run: npm run prebuild
       - name: Type Check
         run: npm run tsc
+      - name: Run ESLint
+        run: npx eslint --ext .ts,tsx src
```

```suggestion
        run: npx eslint --ext .ts,.tsx src
```

I wonder it is a correct usage of *--ext*. When seeing [the document of the option][eslint-ext-link], it used with *dot* prefix for each extension. Of course, it works also in `ts,tsx` and `.ts,tsx`. But I believe it will be better to follow the usage in the document.

[eslint-ext-link]: https://eslint.org/docs/user-guide/command-line-interface#-ext